server/boot/livereload.js

```diff
@@ -0,0 +1,8 @@
+module.exports = function(app) {
+  var livereload = app.get('livereload');
+  if (!livereload) return;
+
+  app.middleware('initial:before', require('connect-livereload')({
+    port: livereload
+  }));
```

Comment:
> ```diff
- // it's important to register the livereload middleware
- // after any response-processing middleware like compress,
- // but before any middleware serving actual content
```

Per that comment to-be-removed, I think `routes:before` is a more appropriate phase for this middleware. Or at least `initial:after`.